Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -14,17 +15,93 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// A button that can be pushed via direct touch.
     /// You can use <see cref="Microsoft.MixedReality.Toolkit.Examples.Demos.PhysicalPressEventRouter"/> to route these events to <see cref="Interactable"/>.
     ///</summary>
+    [InitializeOnLoad]
     [RequireComponent(typeof(BoxCollider))]
     public class PressableButton : MonoBehaviour, IMixedRealityTouchHandler
     {
         const string InitialMarkerTransformName = "Initial Marker";
 
+        #region Experimental Versionining 
```

Comment:
Hm...this versioning stuff is interesting, but is it possible to move somewhere outside of the button class, so the button class can be focused on just that, pressing a button? Otherwise if we add this it's another thing we need to keep in our heads when using buttons, and coudl set a precedent for people putting unrelated code into classes.

Also, regarding versioning....is it required for this PR, or could we do the versioning experiment later? It would be too bad to prevent a bug fix from getting in just because we are experimenting with versioning.

Reply:
> Isn't there a bug that this is actually fixing? It's not mentioned in the PR.

Yes there's at least 2 tickets that are fixed with this PR (https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4219, https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4216) - but I just wanted to open this as a draft for  having a discussion about it. If possible I'd like to avoid having the migration step (if we have to keep it in it should be moved out of the class). 
Lars had a good idea by having a flag that would allow users to switch the mode between world and local space coordinates. that way we wouldn't break anything that is already set up and customers using scaled versions of their buttons can just switch to the new mode where the distances are in local space